src/zarr/store/remote.py

```diff
@@ -71,10 +76,10 @@ def __init__(
             raise TypeError("FileSystem needs to support async operations")
 
     def __str__(self) -> str:
-        return f"Remote fsspec store: {type(self._fs).__name__} , {self.path}"
+        return f"{self._url}"
```

Comment:
We are implying that this store is the only way to access the URL displayed here

Reply:
yes, but this is consistent with our current practice for [memory store](https://github.com/zarr-developers/zarr-python/blob/f648b619e53cde809cd516f308ff40f206a742bd/src/zarr/store/memory.py#L27) and [local store](https://github.com/zarr-developers/zarr-python/blob/f648b619e53cde809cd516f308ff40f206a742bd/src/zarr/store/local.py#L83). Yes, this practice does mean that `RemoteStore` for local files will have the same __str__ output as `LocalStore`, but I figure we should make the stores consistent before making them correct.